components/ws-manager-bridge/src/bridge-controller.ts

```diff
@@ -90,7 +90,7 @@ export class BridgeController {
                 const bridge = await this.createAndStartBridge(newCluster);
                 this.bridges.set(newCluster.name, bridge);
             }
-            log.info("done reconciling.", { allClusters: Array.from(allClusters.values()) });
+            log.info("done reconciling.", { allClusters: Array.from(allClusters.keys()) });
```

Comment:
```suggestion
            log.info("Done reconciling.", { allClusters: Array.from(allClusters.keys()) });
```
Logs tend to be nicer when written for humans - with first word capitalized. But up to you, I don't mind :)

Reply:
> Logs tend to be nicer when written for humans - with first word capitalized

My strong personal preference as well - but as we do not have a guideline for this I don't feel it's relevant. :slightly_smiling_face: